lighthouse-core/report/html/renderer/performance-category-renderer.js

```diff
@@ -182,6 +182,7 @@ class PerformanceCategoryRenderer extends CategoryRenderer {
     const metricsBoxesEl = this.dom.createChildOf(metricAuditsEl, 'div', 'lh-metrics-container');
 
     metricAudits.forEach(item => {
+      if (item.result.scoreDisplayMode === 'notApplicable') return;
```

Comment:
does this not cause other weird things? or since `scoreDisplayMode: 'error'` also has null scores we already handle it ok?

Reply:
I'm not sure what you mean. This line just hides the notApplicable metrics from the metrics section of the category.

Or were you asking if metrics with notApplicable work with the score? If so, the answer is yes, just like others their weight just disappears (though separate weights for timespan is completely different bag of worms!)

Reply:
> Or were you asking if metrics with notApplicable work with the score? If so, the answer is yes, just like others their weight just disappears (though separate weights for timespan is completely different bag of worms!)

yes, I was surprised `notApplicable` just works because we've never had not applicable metrics before, but it was a pleasant surprise :)